omero/training-server/playbook.yml

```diff
@@ -292,7 +292,10 @@
     omero_webtagging_tagsearch_release: "{{ omero_webtagging_tagsearch_release_override | default('3.0.3') }}"
     omero_cli_duplicate_release: "{{ omero_cli_duplicate_release_override | default('0.2.0') }}"
     omero_cli_render_release: "{{ omero_cli_render_release_override | default('0.3.0') }}"
-    os_system_users_password: "{{ os_systems_users_password_override | default('$6$sjH0EeRKZdWYc1y8$F79INhPIfwVUfZ/DRrk.rYEetacoxoqxMVSVRavjggro84vpCqcxUhcNs79C5y4RmdLBb/3sA3vRagfNVjMxD.') }}"
+    # The os_system_users_password default is "ome".
+    # You might wish to override it by a private variable on your system,
```

Comment:
Would you mind changing the wording to something like "You may wish to change this variable, or override it with a private variable os_system_users_password_override"